### PR TITLE
Bugfix/view corrections

### PR DESCRIPTION
**Pull request by AnnaMBuchman**
1. move record section to top
2. change proportions of record img
3. add file section with check icon
4. update menu
5. decrease section paddings 
6. move narrow down button to record section, after click it moves to music options
obraz

### CODE REVIEW

**Review by JakubBilski — CHANGES_REQUESTED** [review]
I don't like the green check thing, I feel it would be more clear without it.
Also, the layout of the whole page feels centered. How about centering this, too?
image
(I also changed the font weights to make "Uploaded file" similar to "Get file" from the recording section)
Instead of writing "Uploaded file: no file selected!", I would say it's better to put "No file selected". It's closer to natural language and feel more elegant.
image

File: view-app/src/css/templatemo-breezed.css [hunk 1]
```diff
@@ -1062,10 +1062,22 @@ ul.slick-dots {
 {
   margin: 0 auto;
 }
+/*file_cestion*/
```
Comment (JakubBilski) [inline view-app/src/css/templatemo-breezed.css:1065]: /*file_section*/

File: view-app/src/css/templatemo-breezed.css [hunk 2]
```diff
@@ -1062,10 +1062,22 @@ ul.slick-dots {
...
+#file_section{  
+  padding: 50px 0px 0px 0px;
+  position: relative;
+  z-index: 9;
+}
+/*choose_cestion*/
```
Comment (JakubBilski) [inline view-app/src/css/templatemo-breezed.css:1071]: section

**Comment by AnnaMBuchman** [comment]
I think it looks better now ;)

**Comment by JakubBilski** [comment]
Now = on my screens? Should I apply my changes here?

**Comment by JakubBilski** [comment]
@JakubBrojacz what do you think?

**Comment by JakubBrojacz** [comment]
The only thing I would change is that for someone who opens our app for the first time text "No file selected" is strange. I thing it would be better to just show "about us"

**Comment by JakubBrojacz** [comment]
We could also remove 
obraz
this thing

**Comment by JakubBilski** [comment]
Ok, I agree

**Review by JakubBilski — APPROVED** [review]

**Pull request merged**